### ISSUE TITLE
MP info with Python 3.8.5

### ISSUE THREAD

**Issue by tsmith2** [brinckmann/montepython_public#185]
I recently upgraded to Python 3.8.5 and when I run info I get the following error:

ValueError: The number of FixedLocator locations (3), usually from a call to set_ticks, does not match the number of ticklabels (1).

and the error points to analyze.py in line 989: ax2dsub.set_xticklabels(['']) .

**Comment by dchooper** [comment]
Hi,

Yesterday we released a new version of MP with a lot of bugfixes, especially for Python 3. Can you try running on the latest version and see if the problem still persists?

Best,
Deanna

**Comment by tsmith2** [comment]
Unfortunately the problem with the error is still there with MP 3.4.0 and python 3.8.5

**Comment by dchooper** [comment]
Hi,

Does this error message appear when you're including an extra plot file, or anytime you are running in info mode? If you are including an extra plot file, what options are you using?

Cheers,
Deanna

**Comment by tsmith2** [comment]
Yes, just when computing the histograms and plotting. I was able to get it to work by downgrading to matplotlib V 3.2.1; the Conda 3.8.5 distribution included matplotlib V 3.3.2.

**Comment by dchooper** [comment]
Ahh, then this problem is the same as the one reported in issue #154.

We do list two possible bugfixes for this in that issue, but it seems these didn't make it into the release last week. We'll propagate this bugfix to the public version in the next minor release.

**Comment by snesseris** [comment]
Hi @dchooper , just to mention I also have the same issue and the solutions in issue #154 do not seem to work for me :(

I have Python 3.8.8 and Matplotlib 3.3.3
